src/Input.js

```diff
@@ -5,7 +5,7 @@ import deprecationWarning from './utils/deprecationWarning';
 
 class Input extends InputBase {
   render() {
-    if (this.props.type === 'static') { // eslint-disable-line react/prop-types
+    if (this.props.type === 'static') {
```

Comment:
I fixed it by https://github.com/react-bootstrap/react-bootstrap/pull/972 some time ago